packages/Button/src/LinkButton.js

```diff
@@ -40,9 +39,8 @@ const LinkButton = React.forwardRef((props, ref) => {
   };
 
   return (
-    <a
+    <Button
```

Comment:
Shouldn't this one be an anchor link?

Reply:
Yes, I think you are correct, I'll make it a styled component that uses an anchor tag